buildSrc/src/main/java/com/r3/csde/CordaLifeCycleHelper.java

```diff
@@ -24,7 +24,11 @@ public CordaLifeCycleHelper(ProjectContext _pc) {
     }
 
     public void startCorda() throws IOException {
-        PrintStream pidStore = new PrintStream(new FileOutputStream(pc.cordaPidCache));
+        File cordaPIDFile = new File(pc.cordaPidCache);
+        if (cordaPIDFile.exists()) {
+            throw new IOException("Cannot start the Combined worker\nCached process ID file " + cordaPIDFile + " existing.\nWas the combined worker already started?");
```

Comment:
I prefer exception strings (explanations) not to contain `\n`

Also I think string interpolation is better than concat:
```suggestion
            throw new IOException("Cannot start the Combined worker. Cached process ID file ${cordaPIDFile} existing. Was the combined worker already started?");
```

Reply:
I have used the same style and wording as in "stopCorda()". See line 94 in the same class. I can change it, but if so we should do it for both startCorda and stopCorda, if that's OK?

Reply:
I agree ... thanks :) 
@mattbradburyr3 FYI

Reply:
both messages are fixed
on "string interpolation is better than concat" this is Java code, and con cat here is not out of the box. Should I go with StringFormatter or we can leave it as it is?

Reply:
No leave as is ;) 
Thanks